views/agentHome.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>agentHomePage</title>
+    <link rel="stylesheet" href="/public/stylesheets/agentHome.css">
+</head>
+<body>
+    <h1>Your Projects</h1>
+    <div id="navbar">
+        <a href="#home">Home</a>
+        <a href="#news">Projects</a>
+        <a href="#contact">Contact</a>
+    </div>
+    <div style="padding:15px 15px 2500px;font-size:30px">
+        <p>Put an image(button) of project here.</p>
+        <p>Another one here</p>
+        <p>And another</p>
+        <p><img src="/views/Pics/addbutton.png" alt=""></p>
```

Comment:
Sorry there was supposed to be an images folder in the public directory. Can u just move ur images into `public/images/example-name.png`

And to link it just do `/images/example-name.png`